### ISSUE TITLE
Build instructions missing

### ISSUE THREAD

**Issue by crshrprt** [bruno-unna/euclidean-rhythms#6]
Thank you for sharing this interesting plugin.

I couldn't find build instructions. Since I've already built some other plugins and programs using meson I managed to build your plugin. However, would be nice to add building instructions to the README, especially for those not familiar with building with meson.

Would be also nice to have instructions on how to install it on a user PATH instead of a system one (currently in /usr/local/lib/x86_64-linux-gnu/). Some programs may even not look for plugins there.

**Comment by bruno-unna** [comment]
Thanks for your interest, @crshrprt.

You are absolutely right. I'll add instructions over the weekend.

In advance, and regarding the installation path, let me tell you that I’ve added an environment file to the project, for meson to use. It's still in the `develop` branch, but if you are happy with that, checkout that branch and you can configure meson this way:
```
meson setup --reconfigure --native-file=environments/linux64.ini builddir
```
or whatever the name of your build directory is.

If you prefer to keep on the released version, which didn't have this environment file yet, instead of the above you could add the option `--libdir=/usr/local/lib` to meson setup.

Thanks for your observations. Kind regards.

**Comment by bruno-unna** [comment]
Done. Released v0.1.1 with the changes.
